src/styles/blocks/card.scss

```diff
@@ -0,0 +1,107 @@
+.card {
+  box-sizing: border-box;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  display: flex;
+  justify-content: center;
+  padding: 32px 16px 16px;
+  -webkit-border-radius: 5px;
+  -moz-border-radius: 5px;
+  -ms-border-radius: 5px;
+  -o-border-radius: 5px;
+
+  &:hover .card__button {
+    box-sizing: border-box;
+    background-color: #fff;
+    border: 1px solid $color-blue;
+    color: $color-blue;
+  }
+}
+
+.card__wrapper {
+  width: 167px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
+
+.card__link-title {
+  @include font-properties(500, 12px, 18px, $color-primary);
+
+  text-align: start;
+  margin-bottom: 4px;
+}
+
+.card__product-code {
+  @include font-properties(400, 10px, 14px, $color-secondary);
+
+  margin: 0 0 16px;
+  padding: 0;
+  width: 100%;
+  text-align: start;
+}
+
+.card__review {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  margin: 0 0 24px;
+}
+
+.card__review-text {
+  display: flex;
+  align-items: center;
+  justify-content: flex-end;
+
+  @include font-properties(400, 10px, 14px, $color-primary);
+
+  padding-top: 2px;
+}
+
+.card__price {
+  margin-bottom: 16px;
+}
+
+.price {
```

Comment:
if button and price are separate BEM blocks they should be written in separate files. Otherwise just make them elements of card block